src/scripts/main.js

```diff
@@ -1 +1,18 @@
 'use strict';
+
+document.addEventListener('DOMContentLoaded', () => {
+  const gallery = document.querySelector('.gallery');
+  const mainImage = document.querySelector('.main-image img');
+
+  gallery.addEventListener('click', (e) => {
+    let target = event.target;
```

Comment:
There is a mistake here: `event` is not defined in this scope. You should use `e` instead, which is the parameter of the event listener callback function.